src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -469,4 +477,42 @@ private String evaluateMacro(String template) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    private void pullRequestActionable(List<PotentialAction> paList, List<Fact> factsList) {
+        Job job = run.getParent();
+        ItemGroup parent = job.getParent();
+        if (parent instanceof MultiBranchProject) {
+            BranchProjectFactory projectFactory = ((MultiBranchProject) parent).getProjectFactory();
+            if (projectFactory.isProject(job)) {
+                Branch branch = projectFactory.getBranch(job);
+                SCMHead head = branch.getHead();
+
+                if (head instanceof ChangeRequestSCMHead) {
+                    String pronoun = StringUtils.defaultIfBlank(head.getPronoun(), "Change Request");
+                    String viewName = String.format("View %s", pronoun);
+                    String titleName = String.format("%s Title", pronoun);
+                    String authorName = String.format("%s Author", pronoun);
+
+                    ObjectMetadataAction oma = branch.getAction(ObjectMetadataAction.class);
+                    if (oma != null) {
+                        String urlString = oma.getObjectUrl();
+                        PotentialAction viewPRPotentialAction = new PotentialAction(viewName, urlString);
+                        paList.add(viewPRPotentialAction);
+                        factsList.add(new Fact(titleName, oma.getObjectDisplayName()));
+                    }
+                    ContributorMetadataAction cma = branch.getAction(ContributorMetadataAction.class);
+                    if (cma != null) {
+                        String contributor = cma.getContributor();
+                        String contributorDisplayName = cma.getContributorDisplayName();
+                        String author = StringUtils.defaultIfBlank(cma.getContributor(), cma.getContributorDisplayName());
+                        if (StringUtils.isNotBlank(contributor) && StringUtils.isNotBlank(contributorDisplayName))
+                            author = String.format("%s (%s)", cma.getContributor(), cma.getContributorDisplayName());
+
+                        if (StringUtils.isNotBlank(author))
```

Comment:
I believe this is always true because of the bracket in string format

Reply:
No you have to read it differently 👍 
It's set to either contributor or contributorDisplayName at the start.
Then we check for if BOTH of those are not blank then we combine them.
If however both of them are blank... just don't add it.

So first `if` checks if BOTH a not blank, then use both.
second checks if BOTH are blank, then removes it.

Reply:
It made sense in my head :bowtie:

Reply:
The brackets won't be used if either one is blank. But if both are blanks we don't show author in the fact list. Hopefully this one makes more sense.